src/Quartz/NetCoreStubs/TypeFilterLevel.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Provide TypeFilterLevel enum values for use on .NET Core (they aren't used by the .NET Framework on Core, but
+// will keep public surface area the same between desktop and Core builds of Quartz.Net).
+// 
+// When running on a platform that already exposes the enum (desktop .NET, for example), redirect references
+// to the actual implementation.
+
+#if REMOTING
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.Serialization.Formatters.TypeFilterLevel))]
+#else // REMOTING
+namespace System.Runtime.Serialization.Formatters
+{
+    //
+    // Summary:
+    //     Specifies the level of automatic deserialization for .NET Framework remoting.
+    public enum TypeFilterLevel
```

Comment:
Yes, we should be able to remove this once remoting support is gone. For the moment I needed it because RemotingSchedulerExporter has a public member (TypeFilterLevel) of this type and I didn't want to change public surface area until you'd given input on the sorts of changes you were comfortable with.